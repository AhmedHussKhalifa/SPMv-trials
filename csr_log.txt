149x149) batch	1	density	0.1	im2col	0.00495527	csr	0.00602004	percent	21.4877
149x149) batch	1	density	0.15	im2col	0.00495306	csr	0.00881852	percent	78.0417
149x149) batch	1	density	0.2	im2col	0.00494532	csr	0.0113396	percent	129.3
149x149) batch	1	density	0.25	im2col	0.00496446	csr	0.013737	percent	176.708
149x149) batch	1	density	0.3	im2col	0.00494302	csr	0.0163406	percent	230.579
149x149) batch	1	density	0.35	im2col	0.00494667	csr	0.0185749	percent	275.503
149x149) batch	1	density	0.4	im2col	0.0049609	csr	0.0206078	percent	315.404
149x149) batch	1	density	0.45	im2col	0.00495703	csr	0.0227295	percent	358.53
149x149) batch	1	density	0.5	im2col	0.00495892	csr	0.0245551	percent	395.17
149x149) batch	1	density	0.55	im2col	0.00494216	csr	0.0264971	percent	436.144
149x149) batch	1	density	0.6	im2col	0.00495352	csr	0.0280267	percent	465.794
149x149) batch	1	density	0.65	im2col	0.00495978	csr	0.0298852	percent	502.551
149x149) batch	1	density	0.7	im2col	0.00496149	csr	0.0314691	percent	534.268
149x149) batch	1	density	0.75	im2col	0.00495189	csr	0.0330635	percent	567.694
149x149) batch	1	density	0.8	im2col	0.00496838	csr	0.034355	percent	591.473
149x149) batch	1	density	0.85	im2col	0.00494505	csr	0.035495	percent	617.788
149x149) batch	1	density	0.9	im2col	0.00495734	csr	0.0368469	percent	643.279
149x149) batch	1	density	0.95	im2col	0.00496068	csr	0.0380966	percent	667.971
149x149) batch	1	density	1	im2col	0.0049605	csr	0.0394019	percent	694.313
149x149) batch	1	density	1.05	im2col	0.00495541	csr	0.0402559	percent	712.363
