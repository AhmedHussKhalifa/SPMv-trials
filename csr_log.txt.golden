
# Node 3 - Mixed 8 
7x1 | 17x17) batch	1	density	0.05	im2col	0.368543	csr	0.151322	percent	58.9406
7x1 | 17x17) batch	1	density	0.1	im2col	0.368	csr	0.287564	percent	21.8576
7x1 | 17x17) batch	1	density	0.15	im2col	0.373042	csr	0.485744	percent	-30.2118
7x1 | 17x17) batch	1	density	0.2	im2col	0.373747	csr	0.618872	percent	-65.5856
7x1 | 17x17) batch	1	density	0.25	im2col	0.374768	csr	0.717474	percent	-91.4448
7x1 | 17x17) batch	1	density	0.3	im2col	0.374225	csr	0.828543	percent	-121.402
7x1 | 17x17) batch	1	density	0.35	im2col	0.376028	csr	0.865931	percent	-130.284
7x1 | 17x17) batch	1	density	0.4	im2col	0.376266	csr	0.96683	percent	-156.954
7x1 | 17x17) batch	1	density	0.45	im2col	0.37373	csr	1.11637	percent	-198.709
7x1 | 17x17) batch	1	density	0.5	im2col	0.377349	csr	1.26287	percent	-234.667
7x1 | 17x17) batch	1	density	0.55	im2col	0.375401	csr	1.27997	percent	-240.961
7x1 | 17x17) batch	1	density	0.6	im2col	0.378394	csr	1.43601	percent	-279.5
7x1 | 17x17) batch	1	density	0.65	im2col	0.373779	csr	1.44558	percent	-286.748
7x1 | 17x17) batch	1	density	0.7	im2col	0.373173	csr	1.51878	percent	-306.99
7x1 | 17x17) batch	1	density	0.75	im2col	0.377952	csr	1.56641	percent	-314.448
7x1 | 17x17) batch	1	density	0.8	im2col	0.371028	csr	1.59877	percent	-330.904
7x1 | 17x17) batch	1	density	0.85	im2col	0.371156	csr	1.72725	percent	-365.371
7x1 | 17x17) batch	1	density	0.9	im2col	0.371526	csr	1.80663	percent	-386.274
7x1 | 17x17) batch	1	density	0.95	im2col	0.375284	csr	1.70157	percent	-353.409
7x1 | 17x17) batch	1	density	1	im2col	0.375114	csr	1.83144	percent	-388.236
7x1 | 17x17) batch	1	density	1.05	im2col	0.378356	csr	1.75948	percent	-365.034
7x1 | 17x17) batch	1	density	0.12	im2col	0.411522	csr	0.390107	percent	5.2039


# Node 3 - Mixed 0
3x3 | 149x149) batch	1	density	0.05	im2col	0.00496446	csr	0.00315603	percent	36.4276
3x3 | 149x149) batch	1	density	0.1	im2col	0.00494825	csr	0.00609283	percent	-23.1312
3x3 | 149x149) batch	1	density	0.15	im2col	0.00494658	csr	0.00877352	percent	-77.3655
3x3 | 149x149) batch	1	density	0.2	im2col	0.00494757	csr	0.0114116	percent	-130.65
3x3 | 149x149) batch	1	density	0.25	im2col	0.00495978	csr	0.0138475	percent	-179.196
3x3 | 149x149) batch	1	density	0.3	im2col	0.00500216	csr	0.016302	percent	-225.899
3x3 | 149x149) batch	1	density	0.35	im2col	0.0049714	csr	0.0185443	percent	-273.02
3x3 | 149x149) batch	1	density	0.4	im2col	0.00495698	csr	0.0206566	percent	-316.718
3x3 | 149x149) batch	1	density	0.45	im2col	0.00495239	csr	0.0226955	percent	-358.273
3x3 | 149x149) batch	1	density	0.5	im2col	0.00498135	csr	0.0244741	percent	-391.315
3x3 | 149x149) batch	1	density	0.55	im2col	0.00495325	csr	0.0265472	percent	-435.956
3x3 | 149x149) batch	1	density	0.6	im2col	0.00494865	csr	0.0279986	percent	-465.782
3x3 | 149x149) batch	1	density	0.65	im2col	0.00497451	csr	0.0299881	percent	-502.835
3x3 | 149x149) batch	1	density	0.7	im2col	0.00499284	csr	0.0314602	percent	-530.108
3x3 | 149x149) batch	1	density	0.75	im2col	0.00495847	csr	0.0328731	percent	-562.969
3x3 | 149x149) batch	1	density	0.8	im2col	0.0049582	csr	0.034328	percent	-592.349
3x3 | 149x149) batch	1	density	0.85	im2col	0.00494933	csr	0.0355194	percent	-617.661
3x3 | 149x149) batch	1	density	0.9	im2col	0.00496315	csr	0.0368937	percent	-643.352
3x3 | 149x149) batch	1	density	0.95	im2col	0.00495361	csr	0.0380481	percent	-668.089
3x3 | 149x149) batch	1	density	1	im2col	0.00496802	csr	0.0394252	percent	-693.58
3x3 | 149x149) batch	1	density	1.05	im2col	0.00498496	csr	0.0402494	percent	-707.418


